Project.toml

```diff
@@ -17,6 +18,7 @@ TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
+ArrowTypes = "1.1"
```

Comment:
The Arrow.jl package should only requires ArrowTypes.jl 1.0 but the tests require 1.1.